internal/distro/rhel8/distro.go

```diff
@@ -678,63 +591,97 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 
 // New creates a new distro object, defining the supported architectures and image types
 func New() distro.Distro {
-	return newDistro("rhel-8")
+	// default minor: create default minor version (current GA) and rename it
+	d := newDistro("rhel", 6)
```

Comment:
Current GA is 8.7, I wonder if that's important though. I must admit that the code smells a bit. :sweat_smile:

Reply:
I fixed the 9 -> 9.1 in a followup commit but not this one.  I'll fix it up.

I'm not too happy about the "initialise and rename" either.  I'm trying to make it easy to have an unversioned constructor point to the proper release without that decision being buried deep in the distro code.  Maybe it belongs in the distro registry instead?

Reply:
Fixed.  But I added it in a new commit instead of squashing into this one to make the change stand out on its own.
I also changed the repo definitions for tests so that rhel-8 == rhel-87, both in `test/data/manifests/` and `test-case-generators/`.